### PR TITLE
Trade log

### PR DESCRIPTION
**Pull request by brittcyr**
Append to a file when running the analysis

Plan is to show this on the liquidity tab to see daily trading summary on the status page

### CODE REVIEW

File: trading_report/src/main.ts [hunk 1]
```diff
@@ -106,7 +106,19 @@ async function main() {
     }
   });
 
-  // TODO: Write out a report of trading
+  appendFile('trade_log.txt', `${SYMBOL} Trading at ${(new Date()).toISOString()}`, () => {});
+  appendFile('trade_log.txt', `
+Openbook trades
+  Bought ${opbTotalBuysAmount} for ${opbTotalBuysValue / (opbTotalBuysAmount + .0000000001)}, net notional: ${opbTotalBuysValue}
```
Comment (DonDuala) [inline trading_report/src/main.ts:112]: Why the + 0.000000001?

**Review by DonDuala — COMMENTED** [review]
BONK rate limited
image

SOL stuck repeatedly "Fetching more signatures"

MNGO looks good

File: trading_report/src/main.ts (hunk 1 above)
Comment (brittcyr) [reply trading_report/src/main.ts:112]: Avoids divide by zero errors

**Comment by brittcyr** [comment]
> BONK rate limited image
> 
> SOL stuck repeatedly "Fetching more signatures"
> 
> MNGO looks good

Not sure it is stuck and isnt just that it is reading through all signatures for SOL for the past day which is a lot. Really should be running a geyser to do this for more popular markets

**Comment by brittcyr** [comment]
Limited number of signatures which should address SOL

**Review by DonDuala — APPROVED** [review]

File: trading_report/trade_log.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+SOL Trading at 2023-01-20T16:56:28.323Z
```
Comment (DonDuala) [inline trading_report/trade_log.txt:1]: remove this or will it just be overwritten all the time?

Comment (brittcyr) [reply trading_report/trade_log.txt:1]: Planning to reset the file in the github action

**Pull request merged**